.travis.settings.xml

```diff
@@ -0,0 +1,9 @@
+<settings>
+    <servers>
+        <server>
+            <id>alfresco-public</id>
```

Comment:
The id of the server does not match the id in the pom's distributionManagement config. I believe the release job will not work.